src/app/services/project_service.js

```diff
@@ -225,6 +227,28 @@ angular
                 // It should not be possible to search for hidden documentation
                 return !obj.docs || obj.docs.show;
             });
+
+			const fuseOptions = {
+			  includeScore: true,
+			  includeMatches: true,
+			  ignoreLocation: true,
+			  //TODO: respect tickboxes for filtering to specific types
+			  keys: [
+			  	{name: 'alias', weight: 20},
+			  	{name: 'tags', weight: 5},
+			  	{name: 'description', weight: 2},
+			  	{name: 'columns.tags', weight: 3},
+			  	{name: 'columns.name', weight: 2},
+			  	{name: 'columns.description', weight: 1},
+			  	{name: 'raw_sql', weight: 1}
```

Comment:
I'm not happy with these weightings yet, but will keep experimenting. 

A potentially controversial opinion is to downweight column names and descriptions so much. I found that in our project, where we use quite verbose column names, having a small score * a large number of columns is still over-representing the column names beyond what I'd like.